domain/src/test/java/org/oppia/android/domain/audio/AudioPlayerControllerTest.kt

```diff
@@ -443,6 +443,18 @@ class AudioPlayerControllerTest {
       .contains("Media player has not been previously initialized")
   }
 
+  @Test
+  fun testController_consecutively_releasePlayer_callDoesNot_throwException() {
+    setUpMediaReadyApplication()
+    audioPlayerController.initializeMediaPlayer()
+
+    // Call releaseMediaPlayer() the first time and assert no exception is thrown
+    audioPlayerController.releaseMediaPlayer()
+
+    // Call releaseMediaPlayer() the second time and assert no exception is thrown
+    audioPlayerController.releaseMediaPlayer()
+  }
```

Comment:
Thanks @adhiamboperes . I
t's not an explicit way to reproduce, but when we play media and a network problem occurs, it triggers setOnErrorListener() in setMediaPlayerListener() in AudioPlayerController.kt. At that time, if we try to press the back button, in some rare cases, releaseMediaPlayer is called multiple times.
Should i add this in issue body?

I am adding the assertion check in test function.

Reply:
Hi @adhiamboperes , could you please check if I can do it this way? Because releaseMediaPlayer() method returns Kotlin.Unit type when no exception occurs.

var exception: Exception? = null
    try {
      audioPlayerController.releaseMediaPlayer()
      audioPlayerController.releaseMediaPlayer()
    } catch (e: IllegalStateException) {
      exception = e
    }

    // Assert that no exception was thrown
    assertThat(exception).isEqualTo(null)

Reply:
Here is a good example to modify your suggestion, by creating a helper method to check that the exception was not thrown: https://www.baeldung.com/junit-assert-exception#2-assert-no-exception-is-thrown-1

Reply:
Thanks